certified-connectors/Connect Bridge/apiDefinition.swagger.json

```diff
@@ -358,6 +394,18 @@
         "summary": "Get next result",
         "description": "This action gets the next result when running multiple queries at once",
         "parameters": [
+          {
+            "name": "format",
+            "in": "query",
+            "required": true,
```

Comment:
Please confirm that the existing flows for this connector will not break from this change.

The required parameter with default value has been added. ValuePath: operation/GetNextResult/parameter/format Source

Reply:
I confirm that the existing flows will not break from this change.

Actually this is a workaround because the policy stopped working for some reason.